Sources/ScipioKit/BuildOptions.swift

```diff
@@ -4,20 +4,17 @@ import OrderedCollections
 struct BuildOptions: Hashable, Codable {
     init(
         buildConfiguration: BuildConfiguration,
-        isSimulatorSupported: Bool,
         isDebugSymbolsEmbedded: Bool,
         frameworkType: FrameworkType,
         sdks: OrderedSet<SDK>
     ) {
         self.buildConfiguration = buildConfiguration
-        self.isSimulatorSupported = isSimulatorSupported
```

Comment:
VersionFiles became to only contain `sdks`. So this flag is removed from VersionFile.

Reply:
Is the VersionFile this BuildOptions.swift?

Reply:
Yes. Since this PR, VersionFile doesn't contain `isSimulatorSupported`. because it is included in `sdks`.